git_machete/client.py

```diff
@@ -1765,7 +1767,7 @@ def create_github_pr(self, head: str, draft: bool) -> None:
         except MacheteException as e:
             print(e)
             return
-
+        self.flush_caches()
```

Comment:
Looks like another piece to be ultimately sorted out by @BartoszSiwinski 's task #237